opt-may-2016/Nipulpatel/com/assignment7/java/Honda.java

```diff
@@ -0,0 +1,24 @@
+/***************************************************************
+ * @author Nipul								 date:6/28/2016 
+ * project:Assignment 7
+ * Scope : Vehicle->Car->Honda relationship.
+ * Scope change : none
+ * class : Sub class of car
+ ***************************************************************/
+package com.assignment7.java;
+
+public class Honda extends Car {
+    
+	public String CarName = "Honda";

```

Comment:
what about encapsulation, with private properties and public getters and setters?